olcPixelGameEngine.h

```diff
@@ -129,7 +129,7 @@
 
 	Author
 	~~~~~~
-	David Barr, aka javidx9, �OneLoneCoder 2018, 2019, 2020
+	David Barr, aka javidx9, �OneLoneCoder 2018, 2019, 2020

```

Comment:
lol

Reply:
I don't even know why that changed, i didn't edit that line at all.. fixed..